src/guardette/config.py

```diff
@@ -1,18 +1,19 @@
 import os
 
-PSEUDONYMIZE_SALT = "PSEUDONYMIZE_SALT"
-CLIENT_SECRET = "CLIENT_SECRET"
-SECRET_MANAGER = "SECRET_MANAGER"
-PSEUDONYMIZE_EMAIL_DOMAINS_ALLOWLIST = "PSEUDONYMIZE_EMAIL_DOMAINS_ALLOWLIST"
+class ConfigManager:
+    REDACT_TOKEN = "[REDACTED]"
 
+    PSEUDONYMIZE_SALT: str = os.environ.get("PSEUDONYMIZE_SALT", "")
 
-class ConfigManager:
-    redact_token = "[REDACTED]"
+    CLIENT_SECRET: str = os.environ.get("CLIENT_SECRET", "")
+    SECRET_MANAGER: str = os.environ.get("SECRET_MANAGER", "default")
+    PROXY_CLIENT_TIMEOUT_SECS: int = os.environ.get("PROXY_CLIENT_TIMEOUT_SECS", 60)
+    SECRET_MANAGER_CACHE_TTL_SECS: int = os.environ.get("SECRET_MANAGER_CACHE_TTL_SECS", 120)
```

Comment:
We'll need to add `PROXY_CLIENT_TIMEOUT_SECS` and  `SECRET_MANAGER_CACHE_TTL_SECS` to terraform as options they can configure